fed/config.py

```diff
@@ -40,13 +40,19 @@ def cross_silo_messages_max_size(self):
 
 
 class JobConfig:
-    def __init__(self) -> None:
-        pass
+    def __init__(self, raw_bytes: bytes) -> None:
+        self._data = cloudpickle.loads(raw_bytes)
+
+    @property
+    def meta_data(self):
```

Comment:
```suggestion
    def metadata(self):
```
Is it conflict with the keeping word in python?

Reply:
Yes, accidentally missing this, thx for that